### PR TITLE
feat: add homeDir() API method (#10)

### PR DESCRIPTION
**Pull request by PierreDemailly**

### CODE REVIEW

File: src/rc.ts [hunk 1]
```diff
@@ -58,6 +63,13 @@ export function generateDefaultRC(mode: RCGenerationMode | RCGenerationMode[] =
   );
 }
 
+/**
+ * Dedicated directory for NodeSecure to store the configuration in the os HOME directory.
+ */
+export function homeDir(): string {
```
Comment (fraxken) [inline src/rc.ts:69]: Can you remove the MAJ ? `homedir` instead of `homeDir`

**Comment by fraxken** [comment]
Also if you can update README.md and add this new API (under write/read methods).

**Comment by PierreDemailly** [comment]
Mince me suis fail sur le README, j'ajoute dans API

File: README.md [hunk 2]
```diff
@@ -49,6 +49,13 @@ const result = (
 assert.strictEqual(result, void 0);
 ```
 
+Get os HOME configuration directory
+```
+import * as RC from "@nodesecure/rc";
+
+const K_HOME_PATH = RC.homedir();
+```
+
```
Comment (fraxken) [inline README.md:58]: I think you can remove this (just the API below is enough).

**Review by fraxken — CHANGES_REQUESTED** [review]

**Comment by fraxken** [comment]
Everything else LGTM.

**Review by fraxken — APPROVED** [review]

**Comment by fraxken** [comment]
@all-contributors please add @PierreDemailly for code

**Pull request merged**